iothub/device/src/Transport/Amqp/ClientWebSocketTransport.cs

```diff
@@ -433,5 +433,13 @@ private static IAsyncResult ToAsyncResult<TResult>(Task<TResult> task, AsyncCall
 
             return tcs.Task;
         }
+
+        private static void AssertAndThrow(bool condition, string description)
```

Comment:
I don't see the point of this helper method. I'd rather just see the code inline.

Worst case, we should add it to the Argument.cs file.